### ISSUE TITLE
Update circle.yml to use built in yarn

### ISSUE THREAD

**Issue by bennett000** [rangle/rangle-starter#208]
Current `circle.yml` files `npm install -g yarn` but there are official instructions.  This typically save ~20-30 seconds on builds: https://circleci.com/docs/install-and-use-yarn/

**Comment by SethDavenport** [comment]
w00t - this was not available from circle when I started

**Comment by bennett000** [comment]
Yeah even in its current state it's got messy output and is kind of patchy, but I'm glad they did it because it really helped on another project.

We also figured out how to do the AoT build in parallel with the test run which helps reduce the time too.  That would be another card and I wanted to ask you about it first.

**Comment by SethDavenport** [comment]
This is a dupe of #221
